sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/key_client_options.hpp

```diff
@@ -27,6 +27,7 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
     ServiceVersion Version;
     Azure::Core::Http::RetryOptions RetryOptions;
     Azure::Core::Http::TransportPolicyOptions TransportPolicyOptions;
+    Azure::Core::Http::TelemetryPolicyOptions TelemetryPolicyOptions;
```

Comment:
Add documentation for KeyClientOptions and all other public items in this header

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/1668